File: 01.JAVA/DA_Secuencia_Nom_Num.java
import java.util.Scanner;
public class DA_Secuencia_Nom_Num{
    public static void main(String[] args){

        int num = 0;

        Scanner entrada = new Scanner(System.in);

        System.out.print("Escriba un número Entero:  ");
        num = entrada.nextInt();

        if(num < 0){
            num = num*(-1);
        }if(num > 0){
            if (num == 1){
                System.out.print("El número 1 es UNO.");
                }else{
                    if(num == 2){
                        System.out.print("El número 2 es DOS.");
                    }else{
                        if(num == 3){
                            System.out.print("El número 3 es TRES.");
                        }else{
                            if(num == 4){
                                System.out.print("El número 4 es CUATRO.");
                            }else{
                                if(num == 5){
                                    System.out.print("El número " + num + " es CINCO.");
                                }else{
                                    System.out.print("El número es MAYOR QUE CINCO!");
                                }
                            }
                        }
                    }
                }

            }
        



    }
}

File: 02.PYTHON/DA_Secuencia_Nom_Num.py

print("")
num = int(input("Escriba un número Entero del uno al cinco:  "))
if(num < 0):
    num = num*(-1)

if(num == 1):
    print("El número es ", num, "(UNO)")
elif(num == 2):
    print("El número es ", num, "(DOS)")
elif(num == 3):
    print("El número es ", num, "(TRES)")
elif(num == 4):
    print("El número es ", num, "(CUATRO)")
elif(num == 5):
    print("El número es ", num, "(CINCO)")
else:
    print("El número es MAYOR QUE CINCO!")